src/index.js

```diff
@@ -95,6 +95,14 @@ function makeMTProto(envMethods) {
       this.updates = new EventEmitter();
     }
 
+    destroy() {
+      for (const rpc of this.rpcs.values()) {
+        rpc.destroy()
+      }
+
+      this.rpcs.clear()
```

Comment:
Please stick to the style. Add a commas and semicolons

Reply:
fixed here https://github.com/alik0211/mtproto-core/pull/235/commits/185d52e3ffbdc65bcfe7b69d810a0b4a4ff87446